Tests/Microsoft.AppCenter.Crashes.Test.Windows/Utils/ErrorLogHelperTest.cs

```diff
@@ -248,22 +322,66 @@ public void SaveErrorLogFile()
         public void SaveErrorLogFileDoesNotThrow(Type exceptionType)
         {
             // Use reflection to create an exception of the given C# type.
-            var exception = exceptionType.GetConstructor(Type.EmptyTypes).Invoke(null) as System.Exception;
+            var exception = exceptionType.GetConstructor(Type.EmptyTypes)?.Invoke(null) as System.Exception;
             var errorLog = new ManagedErrorLog
             {
                 Id = Guid.NewGuid(),
                 ProcessId = 123
             };
-            var fileName = errorLog.Id + ".json";
-            var serializedErrorLog = LogSerializer.Serialize(errorLog);
             var mockDirectory = Mock.Of<Directory>();
             ErrorLogHelper.Instance._crashesDirectory = mockDirectory;
             Mock.Get(mockDirectory).Setup(d => d.EnumerateFiles(It.IsAny<string>())).Throws(exception);
-            ErrorLogHelper.SaveErrorLogFile(errorLog);
+            ErrorLogHelper.SaveErrorLogFiles(null, errorLog);
 
             // No exception should be thrown.
         }
 
+        private class NonSerializableException : System.Exception
```

Comment:
Is this just here as an exception type that doesn't implement a serialization method?  I would prefer for it to be at the top/bottom just so it's not in the middle of all the test methods